about.html

```diff
@@ -5,24 +5,22 @@
     <title>Straight to the Point Archery - Home</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <link rel="stylesheet" href="./css/styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/910f234e96.js" crossorigin="anonymous"></script>
     <link href="https://fonts.googleapis.com/css2?family=Cairo:wght@400;600&family=Fjalla+One&display=swap" rel="stylesheet">
```

Comment:
Don't forget, the order of imports matters here, and your own custom `styles.css` ought to be the very last <link> tag, otherwise you might get CSS errors because styles can be overwritten or sometimes fonts be unavailable.